.github/workflows/ci.yml

```diff
@@ -100,17 +100,23 @@ jobs:
       - name: Checkout extensions
         uses: actions/checkout@v3
 
-      - uses: actions/setup-go@v4
-        with:
-          go-version: '1.20.4'
-
       - name: Checkout containerd
         uses: actions/checkout@v3
         with:
           repository: containerd/containerd
           path: src/github.com/containerd/containerd
           ref: ${{ matrix.containerd }}
 
+      - name: Get Go Version
+        run: |
+          go_version=$(awk -F': ' '/GO_VERSION/ {gsub(/["'\'']/, "", $2); print $2; exit}' .github/workflows/release.yml)
```

Comment:
We don't have `.github/workflows/release.yml`, right?

```
❯ cat .github/workflows/release.yml
cat: .github/workflows/release.yml: No such file or directory
```

Reply:
this is the containerd release file since it is the containerd repository we checked out in the step before

Reply:
oh, right right.